modules/ROOT/pages/starting-and-stopping-mule-esb.adoc

```diff
@@ -0,0 +1,180 @@
+= Starting and Stopping Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, esb, deploy, launch, run, terminate
+
+[NOTE]
+====
+This section deals with deploying to standalone instances of the Mule Runtime. To take full advantage of managing and monitoring features, you can deploy to the same Mule Runtime via the xref:runtime-manager::index.adoc[Runtime Manager]. See xref:runtime-manager::deployment-strategies.adoc[Deployment Strategies] for a deeper look at the different deployment alternatives offered by the platform.
```

Comment:
1. If possible, remove self-referential content like "This section deals with". Also, the title is starting and stopping, not deploying, so I'm a bit confused.